lib/cc/engine/analyzers/php/main.rb

```diff
@@ -20,7 +20,7 @@ class Main < CC::Engine::Analyzers::Base
           private
 
           def process_file(path)
-            code = File.read(path)
+            code = File.binread(path)
```

Comment:
Makes sense to me. I didn't check the others to see if they were using `read` or `binread`.